article.html

```diff
@@ -7,37 +7,76 @@
+			<nav name="top">
+				<ul>
+					<li><a href="quiz.html">skin type quiz</a></li>
+					<li class="dropdown"><a href="all-products.html">products &#9662;</a>
+						<div class="dropdown-content">
+							<a href="#">oily skin products</a>
+							<a href="#">dry skin products</a>
+							<a href="#">combo skin products</a>
+							<a href="#">sensitive skin products</a>
+						</div>
+					</li>
+					<li class="dropdown"><a href="article-list.html">articles &#9662;</a>
+						<div class="dropdown-content">
+							<a href="#">oily skin articles</a>
+							<a href="#">dry skin articles</a>
+							<a href="#">combo skin articles</a>
+							<a href="#">sensitive skin articles</a>
+						</div>
+					</li>
+				</ul>
+			</nav>
 		</header>
 		
 		<div class="wrapper">
             <!-- This div is for the template for all articles -->
             <div class="article">
                 <h1>Article header here</h1>
-                <h2>Author and publish date here</h2>
+                <h2>Skinclusive | DD/MM/YYYY</h2>
 
                 <div class="video">
                     <p>Placeholder for a video</p>
                 </div>
-                <p>Article text will go here. Relevant images will also be spread out through this page</p>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor 
+                    incididunt ut labore et dolore magna aliqua. Curabitur vitae nunc sed velit dignissim sodales ut.
+                    Blandit massa enim nec dui nunc mattis. Varius quam quisque id diam vel quam elementum.
+                    Dui ut ornare lectus sit amet. Elementum eu facilisis sed odio morbi quis commodo odio. 
+                    Risus sed vulputate odio ut enim blandit volutpat maecenas. Nisi est sit amet facilisis magna. 
+                    At in tellus integer feugiat scelerisque varius morbi enim. Eget lorem dolor sed viverra. 
+                    Aliquet eget sit amet tellus cras. Et malesuada fames ac turpis egestas maecenas pharetra 
+                    convallis posuere. Faucibus et molestie ac feugiat sed. Ipsum suspendisse ultrices gravida 
+                    dictum fusce ut.</p>
+                
+                <div class="content-image">Images will be scattered where relevant</div>
+                
+                <p>Augue ut lectus arcu bibendum at varius vel. Ultricies leo integer malesuada nunc vel risus 
+                    commodo viverra maecenas. Donec ultrices tincidunt arcu non sodales neque sodales. Porta 
+                    lorem mollis aliquam ut. Faucibus in ornare quam viverra orci sagittis. Elementum nisi quis
+                    eleifend quam adipiscing. Id aliquet lectus proin nibh nisl condimentum id venenatis.</p>
+                
             </div>
 
             <!-- This div is for similar articles that will be displayed below -->
             <div class="similar">
+                <hr>

```

Comment:
In my final layout, I decided to add a thin horizontal line to split the article with the similar ones as I thought it looked better rather than without.